geode-core/src/test/java/org/apache/geode/management/internal/cli/shell/GfshJunitTest.java

```diff
@@ -112,4 +118,24 @@ else if (loggerName.endsWith(".SimpleParser")) {
       }
     }
   }
+
+  @Test
+  public void executeCommandShouldSubstituteVariablesWhenNeededAndDelegateToDefaultImplementation() {
```

Comment:
would be good to have another test demonstrate that if user's command has a legitimate $ in there, like "echo --string="I have $50". It would not do any substitution, or if the the  property is not defined in the environment.

Reply:
👍